### DISCUSSION TITLE
Engrampa as default archive manager

### DISCUSSION THREAD

**Discussion by tenplus1** [linuxmint/discussions#522] [Ideas]
File-roller is suddenly having issues in Mint 21.3 and 22 when dragging single files into a zipped folder and bringing up an error message saying it cannot be added to the archive, but it works perfectly when using Engrampa, so...

Wouldn't it be a better idea to ditch file-roller and it's adwaita gui completely and adopt Engrampa for all flavours of Mint which works and uses GTK3 theming to match everything ???

**Comment by billyswong** [comment]
There is a horrible bug in Engrampa 1.26.2 as shipped in LM 22 and LM 22.1. People can't open password protected 7z files.
https://github.com/mate-desktop/engrampa/issues/510
